src/modules/users/test/users.controller.spec.ts

```diff
@@ -108,4 +123,59 @@ describe('UsersController', () => {
       expect(controller.deleteUser('1')).resolves.toBeUndefined();
     });
   });
+
+  describe('wishlistAccommodation', () => {
+    it('should add accommodation to wishlist if not already in', async () => {
+      const userId = mockUsers[0].id;
+      const accommodationId = 'accommodation-2'; // New accommodation ID
+      const mockRequest = { user: { sub: userId } }; // Mocking the request object
+
+      // Mock the service method to resolve successfully
+      jest.spyOn(service, 'wishlistAccommodation').mockResolvedValue(undefined);
```

Comment:
you are already mocking the resolved value to undefined when instantiating the module. why you are mocking it again?